### DISCUSSION TITLE
The ForeignKey is displayed (<models.mls.user.User object at 0x000001F34B5C6560>)

### DISCUSSION THREAD

**Discussion by wnesbv** [aminalaee/sqladmin#102] [Q&A] [answered]
https://github.com/wnesbv/fastapi-web-html

**Comment by aminalaee** [comment, marked as answer]
Please provide more information about your code and describe your question properly.
I think what you need to do is define `__str__` method on your models, that's not really related to SQLAdmin.

**Comment by wnesbv** [comment]
Thank you..! That's right..
